governance/assessments/projects/karmada/2023-08-29.md

```diff
@@ -0,0 +1,140 @@
+
+#### Sub-projects, plugins, and related
+
+The project includes the following sub-projects, plugins, and other notable divisions:
+
+None
+
+### Operation
+
+The community meeting [doc](https://docs.google.com/document/d/1y6YLVC-v7cmVAdbjedoyR5WL0-q45DBRXTvz5_I7bkA/edit#heading=h.g61sgp7w0d0c) has a lively history of meetings and topics. The karmada [slack](https://cloud-native.slack.com/archives/C02MUF8QXUN) channel is active with reponses from the community, but it seems there are less answers in the karmada dev [channel](https://cloud-native.slack.com/archives/C02NDR16M3K).
+
+#### Transparency and freshness
+
+Transparency for a project is exemplified in the public documentation, record, and communications, allowing observers and contributors to monitor the project's adherence to their stated governance. Freshness indicates governance activities mirror the documented governance for the project, and have been reviewed or updated recently.
+
+The project's governance documentation and activities are not currently up to date. The project currently has two different owners files in different repos ([Owners](https://github.com/karmada-io/community/blob/main/OWNERS) and [Owners](https://github.com/karmada-io/karmada/blob/master/OWNERS)) and the community ladder currenly links to a third [Owners](https://github.com/karmada-io/community/blob/main/contributors/guide/owners.md) that is currently empty. 
+
+The [Approvers](https://github.com/karmada-io/community/blob/main/APPROVERS.md) is a relatively new so seems to be up to date while the [Reviewers](https://github.com/karmada-io/community/blob/main/REVIEWERS.md) has not been updated very much since it was created. This is also out of sync with the nubmer listed in the incubation proposal, but it may be because that document was created in December 2022.
+
+There are currently 6 outstanding [requests](https://github.com/karmada-io/community/issues) for community membership, some as old as 7 weeks. Community membership is also not tracked publically so it is hard to tell how many members there are. 
+
+#### Governance Drift
+
+The project does experience governance drift as indicated by inactive maintainers and no new maintainer onboarding.
+
+#### Ownership
+
+The project's ownership evaluation did not leverage Sheriff, the CNCF GitHub permission auditing tool.
+
+The project's permissions and ownership settings and files were not checked.
+
+### Maintainer List(s)
+
+The project's maintainer list is not current. Individuals on the maintainer list do not appear to match the requirements of maintainership in accordance with the project's documented requirements. The maintainer affiliations (employers) reflect Balanced diversity.
+
+@mrlihanbo has not contributed since August 2022
+@carmark has not contributed since August 2021
+@zoroyouxi has no public contributions
+@lfbear has not contributed since December 2021
```

Comment:
for future reviews, it may be beneficial to define what is observable as a contribution? or perhaps where contributions were looked at (GitHub, slack, discussions, meetings, etc.). I know we wont check all the places, but it does set context for what was or wasnt discoverable.

Reply:
Should I create an issue for this?

Reply:
No, we just need to say what data we looked at for contributions.